central/complianceoperator/v2/report/datastore/datastore_impl.go

```diff
@@ -29,3 +32,45 @@ func (d *datastoreImpl) UpsertSnapshot(ctx context.Context, result *storage.Comp
 func (d *datastoreImpl) DeleteSnapshot(ctx context.Context, id string) error {
 	return d.store.Delete(ctx, id)
 }
+
+// DeleteOrphanedReportSnapshots deletes all the snapshots that are not in a final state
+func DeleteOrphanedReportSnapshots(ctx context.Context, ds DataStore) error {
```

Comment:
A comment that this runs only on startup of Central would be helpful

Reply:
Done